static/src/javascripts/projects/commercial/modules/header-bidding/prebid/bid-config.ts

```diff
@@ -449,6 +457,8 @@ const biddersSwitchedOn = (allBidders: PrebidBidder[]): PrebidBidder[] => {
 
 const currentBidders = (slotSizes: HeaderBiddingSize[]): PrebidBidder[] => {
 	const otherBidders: PrebidBidder[] = [
+		// Currently only include Criteo if explicitly using pbtest query parameter
+		...(inPbTestOr(false) ? [criteoBidder] : []),
```

Comment:
This is the crucial part: at this stage the **only** way to have Criteo participate in an auction is to explicitly pass the `?pbtest=criteo` flag. All other auctions will remain unchanged.

Reply:
Non-blocking: reading through `inPbTestOr` it looks like we'd include Criteo if the `pbtest` parameter was set to _any_ value. So `?pbtest=blablabla` would have the same effect. 
It might be worth changing `inPbTestOr` and `isPbTestOn` to take a `bidderName` and return whether that bidder is being tested. (Not necessarily in this PR)

Reply:
It's not actually that clear but I think this check is actually to see if we're using the `pbtest` parameter at all and then a subsequent check is performed by the `biddersBeingTested` function to check if it's supplied in the query string. 

This should work, for example trying `https://theguardian.com/uk?pbtest=improvedigial` or `https://theguardian.com/uk?pbtest=ozone` will give you just those bidders but `https://theguardian.com/uk?pbtest=foobar` will give no bids.

Reply:
Oh good call - I hadn't noticed the filtering happening later! Ignore me